src/client/components/WeekWithDetail/index.js

```diff
@@ -87,22 +89,40 @@ function WeekWithDetail(props) {
     return getSubmittedAnswer(questionName, weekData);
   });
 
+  // TODO(kalvin): refactor collapsible accordion item from here and Acknowledgement
+  // in RetroCertsConfirmationPage out to common file
+  const EN_DASH = "–";
   return (
-    <Alert variant="secondary" className="d-flex">
-      <div className="flex-fill">
-        <Trans
-          t={t}
-          i18nKey="retrocerts-week-list-item"
-          values={{ ...dates, weekForUser }}
-        />
-        <WeekConfirmationDetails
-          employers={weekHasEmployers ? weekData.employers : undefined}
-          questionAnswers={questionAnswers}
-          questionKeys={questionKeys}
-          weekString={toWeekString(weekIndex)}
-        />
-      </div>
-    </Alert>
+    <React.Fragment>
+      <Alert variant="secondary" className="d-flex">
+        <div className="flex-fill">
+          <button
```

Comment:
We probably need to add an aria property for this so screen readers will say something useful. We might need to get that translated to Spanish.

Reply:
I don't think so, since it's a button element which includes aria support from Bootstrap? This is the same as all the other buttons we use, the screen reader will presumably read aloud that it's a button which can be clicked, and the content inside (the week dates)

Reply:
Right, I meant we might want to have it say something like "show details for ..." and "hide details for ...". I'm not sure having it say "plus week two Sunday 03/15/2020 - Saturday 03/21/2020 button" is that useful.

Reply:
Oh, good point! Will add in English. I don't think we support Spanish for any other accessibility strings, like alt text? I'll check with Michelle

Reply:
Yes definitely for "Show details for...." / "Hide details for... "

Reply:
@idealisms it now says "Show details for this week" and "Mostrar detalles para esta semana" as the aria label for the buttons!